handlers/SalesforceAgentAssignedHandler.ts

```diff
@@ -31,15 +31,8 @@ export class SalesforceAgentAssigned {
 		const salesforceAgentName = (await retrievePersistentData(this.read, assoc)).salesforceAgentName;
 		const technicalDifficultyMessage: string = await getAppSettingValue(this.read, AppSettingId.TECHNICAL_DIFFICULTY_MESSAGE);
 
-		let salesforceChatApiEndpoint: string = await getAppSettingValue(this.read, AppSettingId.SALESFORCE_CHAT_API_ENDPOINT);
-		try {
-			salesforceChatApiEndpoint = salesforceChatApiEndpoint.replace(/\/?$/, '/');
-		} catch (error) {
-			await sendLCMessage(this.read, this.modify, this.data.room, technicalDifficultyMessage, this.data.agent);
-			await sendDebugLCMessage(this.read, this.modify, this.data.room, ErrorLogs.SALESFORCE_CHAT_API_NOT_FOUND, this.data.agent);
-			console.error(ErrorLogs.SALESFORCE_CHAT_API_NOT_FOUND, error);
-			return;
```

Comment:
We are not sending technical difficulty message anymore post this change. Is this expected behaviour?

Reply:
Yes, we are not sending technicalDifficultyMessage. It's redundant. In the try-catch we are only doing string replacement. Also, we are not having access to all the parameters of sendMessage fn everywhere.